xwiki-platform-distribution/xwiki-platform-distribution-flavor/xwiki-platform-distribution-flavor-test/xwiki-platform-distribution-flavor-test-security/src/test/it/org/xwiki/test/security/AllTests.java

```diff
@@ -34,6 +38,21 @@
 @RunWith(PageObjectSuite.class)
 public class AllTests
 {
+    @XWikiExecutorSuite.PreStart
+    public void preStart(List<XWikiExecutor> executors) throws Exception
+    {
+        // We need to have access to the maven repositories to be able to installed the 
+        // xwiki-platform-extension-security-ui extension at runtime (from the local repository, or from the remote
+        // final and snapshot repositories).
+        XWikiExecutor executor = executors.get(0);
+        executor.loadXWikiPropertiesConfiguration().setProperty("extension.repositories", List.of(
+            "maven-local:maven:file://${sys:user.home}/.m2/repository",
```

Comment:
I feel like the local repository should be enough. Or is the point to have this test work even if there was no Maven build done before ?

Reply:
Yes, I felt like it could help to have them listed, even though they are not expected to be used by default.

Reply:
List reduced to "maven-local:maven:file://${sys:user.home}/.m2/repository" only